services/app/src/components/base/Dialogs.tsx

```diff
@@ -277,11 +277,11 @@ export class SetPlayerCountDialog extends React.Component<SetPlayerCountDialogPr
     const quest = this.props.quest;
     let playersAllowed = true;
     if (quest.minplayers && quest.maxplayers) {
-      playersAllowed = (this.props.settings.numPlayers >= quest.minplayers &&
-        this.props.settings.numPlayers <= quest.maxplayers);
+      playersAllowed = (this.props.settings.numLocalPlayers >= quest.minplayers &&
+        this.props.settings.numLocalPlayers <= quest.maxplayers);
     }
     let contents = <div>
-        <Picker id="adventurerCount" label="Adventurers" value={this.props.settings.numPlayers} onDelta={(i: number) => this.props.onPlayerDelta(this.props.settings.numPlayers, i)}>
+        <Picker id="adventurerCount" label="Adventurers" value={this.props.settings.numLocalPlayers} onDelta={(i: number) => this.props.onPlayerDelta(this.props.settings.numLocalPlayers, i)}>
```

Comment:
WDYT about also indicating the total player count when multiplayer is connected on this dialog? Here and on the game setup page?

Reply:
That sounds like a good idea, but there's currently a weird delay of a few seconds (even on the local client) of adventurer count changes, so I've added that as a separate issue - https://github.com/ExpeditionRPG/expedition/issues/237